### ISSUE TITLE
Automatic execution of example notebooks

### ISSUE THREAD

**Issue by matthewkirby** [LSSTDESC/CLMM#198]
Currently, rtd automatically converts our examples to webpages.

Automatic execution however is still lacking. Automatic execution can be turned on by simply uncommenting the `--execute` line in `docs/conf.py`.

## Locally...
It works! If the kernal dies, `pip install --upgrade nbconvert` was enough to fix it. 

In the process, it copies ~10-12 figures towards the end of the process.

## On travis...
Sometimes it "works" sometimes it doesn't. I am not entirely sure yet why it's so inconsistent.

However, at the end of the process, when it "works" it only copies 1 figure. I expect that something is amiss for that reason, even when it "works".

## On readthedocs...
Requires configuring a conda environment. I am pretty close to having this running. Current error message is `ModuleNotFoundError: No module named 'clmm'`. I can have conda install the master branch of the repo but I am not sure how to install whatever your working branch is (the branch you are building docs for).

## In conclusion...
If this is not working by the time v1.0 release comes along, we can execute locally and push. IMO, this should be done explicitly for the tagged release version and should not be in master.

**Comment by matthewkirby** [comment]
This is no longer relevant as we have changed how the documentation is hosted.
